packages/react/src/themes/teams/components/Icon/svg/icons/alert.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react'
+import { TeamsProcessedSvgIconSpec } from '../types'
+
+export default {
+  icon: ({ classes }) => (
+    <svg role="presentation" focusable="false" className={classes.svg} viewBox="8 8 16 16">
```

Comment:
@codepretty This icon is always outline, never filled:
![image](https://user-images.githubusercontent.com/9615899/60124096-2e9f0d00-9789-11e9-91f6-cf7e0d6d3f27.png)
Is it ok?

Reply:
For my case is fine. Also, for this particular one I just exported this from the processed ones. I did not create this from scratch.

Reply:
Here is the filled path

`<path class="filled" d="M23.7,20L17.6,9.7c-0.1-0.2-0.3-0.4-0.5-0.5C16.9,9.1,16.7,9,16.4,9c-0.2,0-0.5,0.1-0.7,0.2c-0.2,0.1-0.4,0.3-0.5,0.5 L9.2,20C9.1,20.2,9,20.4,9,20.7c0,0.4,0.1,0.7,0.4,1c0.3,0.3,0.6,0.4,1,0.4h12.1c0.4,0,0.7-0.1,1-0.4c0.3-0.3,0.4-0.6,0.4-1 C23.9,20.4,23.8,20.2,23.7,20z M16,12.6c0-0.3,0.2-0.5,0.5-0.5s0.5,0.2,0.5,0.5v4.7c0,0.3-0.2,0.5-0.5,0.5S16,17.6,16,17.3V12.6z M16.5,20.8c-0.4,0-0.8-0.3-0.8-0.8c0-0.4,0.3-0.8,0.8-0.8s0.8,0.3,0.8,0.8C17.3,20.4,16.9,20.8,16.5,20.8z"></path>`